### DISCUSSION TITLE
How to custom 404 page in NEXTJS 13

### DISCUSSION THREAD

**Discussion by MasterProvider** [vercel/next.js#42774] [Help] [answered]
404 custom page not changing in the app directory

//404.js

```
export default function CustomNotfound() {
  return (
    <div>Custom 404</div>
  )
}
```

how do I implement this in NEXTJS 13?

**Comment by Valefant** [comment, marked as answer]
The app directory supports this: https://beta.nextjs.org/docs/api-reference/file-conventions/not-found

But this it does not seem to work (at least for me).

What you can do instead is the following: Create a folder `[...404]` and place a `page.tsx` file inside. This will act as a catch all route in case a path cannot be matched
